app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,100 @@
+class InvitationsController < ApplicationController
+  before_action :restrict_access_on_create, only: [ :new, :create ]
+  before_action :restrict_access_on_update, only: [:edit, :update]
+
+  # GET /invitations/new
+  def new
+    @user = User.new
+  end
+
+  # POST /invitations
+  def create
+    @user = User.new(invitation_params)
+
+    # Fictive params to save new user
+    @user.update(name: "name", surname: "surname", gender: :male, 
+                 password: "password", password_confirmation: "password")
```

Comment:
`password: "password`? Are you serious? Why don't you rewrite the whole process in a way that you won't need these strange values?